### ISSUE TITLE
Add a Mass control focus alert that describes size of Force vectors

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#143]
Now that aria-valuetext is implemented for the Mass slider values (same as the position slider), please add the same force focus alert that is present on the position sliders.

This alert gives a brief qualitative description of the size of the force vector arrows giving an idea of the size of the force before it starts changing.

This would be common across both GFL REGULAR and BASICS, but the wording would be different for BASICS, "Force arrows {{qualitative-size}}."

Here's an example of the alert after a position slider gets focus and after the value for position has been read out. I'd like the same brief force focus alert when a mass control gains focus.

<img width="516" alt="force-focus-alert" src="https://user-images.githubusercontent.com/1902116/51539207-b9cf4c00-1e53-11e9-977d-f03ce632682a.png">

**Comment by zepumph** [comment]
The wording will be covered in https://github.com/phetsims/gravity-force-lab-basics/issues/86, but the alerts should be working as expected in master now. Over to @terracoda to review.

**Comment by terracoda** [comment]
Hearing and seeing the force vector alerts on mass controls now. Thanks @zepumph! Closing.
